src/main/java/org/opentripplanner/api/resource/LIsochrone.java

```diff
@@ -149,7 +149,7 @@ public Response getZippedShapefileIsochrone(@QueryParam("shpName") String shpNam
      * Create a geotools feature collection from a list of isochrones in the OTPA internal format.
      * Once in a FeatureCollection, they can for example be exported as GeoJSON.
      */
-    public static SimpleFeatureCollection makeContourFeatures(List<IsochroneData> isochrones) {
+    public static synchronized SimpleFeatureCollection makeContourFeatures(List<IsochroneData> isochrones) {
         DefaultFeatureCollection featureCollection = new DefaultFeatureCollection(null,
                 contourSchema);
         SimpleFeatureBuilder fbuilder = new SimpleFeatureBuilder(contourSchema);
```

Comment:
Here each `SimpleFeatureBuilder` instance are only used internally in the method, so it should not be used concurrently in multiple threads; are you sure a `synchronized` is necessary? i.e. are you implying that we can't use *any* `SimpleFeatureBuilder` concurrently?